diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('li').first().parent().parent().children().last();
```

Comment:
Your answer does work in this case, but to be safe, this would be better: `$('li').parent().parent().children('form');`